AdyenCard/Components/Card/CardComponentExtensions.swift

```diff
@@ -107,26 +108,9 @@ extension CardComponent {
     ///   - publicKey: The key used for encrypting card data.
     ///   - style: The Component's UI style.
     @available(*, deprecated, message: "Use init(paymentMethod:clientKey:style:) instead.")
-    public convenience init(paymentMethod: AnyCardPaymentMethod,
-                            publicKey: String,
-                            style: FormComponentStyle = FormComponentStyle()) {
-        let cardPublicKeyProvider = CardPublicKeyProvider(cardPublicKey: publicKey)
-        self.init(paymentMethod: paymentMethod, cardPublicKeyProvider: cardPublicKeyProvider, style: style)
-        if !isPublicKeyValid(key: publicKey) {
-            assertionFailure("Card Public key is invalid, please make sure it’s in the format: {EXPONENT}|{MODULUS}")
-        }
-    }
-    
-    /// Initializes the card component for stored cards.
-    ///
-    /// - Parameters:
-    ///   -  paymentMethod: The stored card payment method.
-    ///   -  publicKey: The key used for encrypting card data.
-    ///   -  style: The Component's UI style.
-    @available(*, deprecated, message: "Use init(paymentMethod:clientKey:style:) instead.")
```

Comment:
Why (only) this deprecated method is removed?
I don't see "deprecated: ***" commit message :P

Reply:
The removed function is a duplicate of another function:

one has this signature:

```
(paymentMethod: AnyCardPaymentMethod, publicKey: String, style: FormComponentStyle = FormComponentStyle())
```

and the other has this signature:

```
(paymentMethod: StoredCardPaymentMethod, publicKey: String, style: FormComponentStyle = FormComponentStyle())
```

and `StoredCardPaymentMethod ` is a `AnyCardPaymentMethod `, so code beat flagged this as code duplication.

Reply:
ok, sounds fair :D